nuget/buildTransitive/uno.fonts.fluent.props

```diff
@@ -3,4 +3,8 @@
 	<PropertyGroup Condition="'$(UnoFontsFluentDisableImport)'==''">
 		<UnoPlatformDefaultSymbolsFontFamily>ms-appx://uno.fonts.fluent/Fonts/uno-fluentui-assets.ttf</UnoPlatformDefaultSymbolsFontFamily>
 	</PropertyGroup>
+
+	<Target Name="_UnoAddPriMarker" AfterTargets="ResolvePackageAssets">
```

Comment:
I wanted to avoid relying on an internal implementation detail of the marker resolution assembly (assuming that most libraries will also contain code). We still have additional work to do on fonts, and this may be part of it to include a more robust way of creating font-only packages.

Reply:
I'm not sure I understand the problem with relying on the internal implementation detail. But anyway, let me revert this part for now.

Reply:
The use of `_UnoPriFiles` is an internal implementation detail at this time.